django/core/management/commands/runserver.py

```diff
@@ -108,22 +108,9 @@ def run(self, **options):
         else:
             self.inner_run(None, **options)
 
-    def inner_run(self, *args, **options):
-        # If an exception was silenced in ManagementUtility.execute in order
-        # to be raised in the child process, raise it now.
-        autoreload.raise_last_exception()
-
-        threading = options['use_threading']
-        # 'shutdown_message' is a stealth option.
-        shutdown_message = options.get('shutdown_message', '')
+    def _on_bind(self, server_name, server_port):
+        """Display server name and port after server bind"""
```

Comment:
Please add a period:
```suggestion
        """Display server name and port after server bind."""
```